gcloud/datastore/helpers.py

```diff
@@ -218,44 +218,6 @@ def _get_value_from_property_pb(property_pb):
     return _get_value_from_value_pb(property_pb.value)
 
 
-def _set_protobuf_property(property_pb, name, value, indexed):
```

Comment:
Right but the functionality is not been used correctly, this partly due to #512 / #404 being caught up in our big move.

However, by removing this, we re-introduce #403.

Calling [`helpers._set_protobuf_value`][1] should be replaced in the `_assign` helper.

[1]: https://github.com/GoogleCloudPlatform/gcloud-python/blob/64aaed4190ca4b0b9457105636d0ec6893e824ca/gcloud/datastore/batch.py#L316